cli/src/config.ts

```diff
@@ -237,6 +237,8 @@ async function loadIOSConfig(
   const platformDirAbs = resolve(rootDir, platformDir);
   const nativeProjectDir = 'App';
   const nativeTargetDir = `${nativeProjectDir}/App`;
+  const nativeXcodeProjDir = `${nativeProjectDir}/App.xcodeproj`;
+  const nativeXcodeWorkspaceDir = `${nativeProjectDir}/App.xcworkspace`;
```

Comment:
```suggestion
  const nativeTargetDir = `${nativeProjectDir}/${nativeProjectDir}`;
  const nativeXcodeProjDir = `${nativeProjectDir}/${nativeProjectDir}.xcodeproj`;
  const nativeXcodeWorkspaceDir = `${nativeProjectDir}/${nativeProjectDir}.xcworkspace`;
```
if we do it like this, would be easier to allow users to rename their project (if we decide to make the `nativeProjectDir` value configurable)